pkg/minikube/constants/constants.go

```diff
@@ -44,7 +43,7 @@ const (
 	DefaultDiskSize = "20g"
 )
 
-var DefaultIsoUrl = "https://github.com/minishift/minishift/releases/download/v" + version.GetVersion() + "/boot2docker.iso"
+var DefaultIsoUrl = "https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.0/minishift-b2d.iso"
```

Comment:
-1 for hard coded the version like this. If we hard code it (which I am ok doing initially), we need to do it in the Makefile (inlining the main Minishift version and the OpenShift version).

If we want to GitHub, we want to define a base version say 1.x and then use semantic versioning to get the latest version which is compatible. Not just the latest version. The functionality is already there to do this. So on sets the base version 1.x (eg in the Makefile). Then you go out to GitHub and get the latest version which is compatible in a semantic versioning sense. This is what I had in mind.

There is no risk to hit the Git Hub API rate limit here. We only do this when the ISO is not cached. There is no automatic update right now.